aws/src/main/java/org/apache/iceberg/aws/s3/S3InputFile.java

```diff
@@ -82,6 +97,27 @@ public long getLength() {
 
   @Override
   public SeekableInputStream newStream() {
+    if (s3FileIOProperties().isS3AnalyticsAcceleratorEnabled()) {
+      LOG.info("Using S3 analytics accelerator stream");
```

Comment:
nit: should combine the separated log lines, and also it's usually not a good practice to print all configs. If you want to do that, it should be at debug level, and I am wondering if you can do that in your library instead of in Iceberg

Reply:
Yeah, I agree. Will remove these logs at this place.